src/main/java/com/algorand/algosdk/v2/client/algod/GetBlock.java

```diff
@@ -0,0 +1,47 @@
+package com.algorand.algosdk.v2.client.algod;
+
+import com.algorand.algosdk.v2.client.common.Client;
+import com.algorand.algosdk.v2.client.common.Query;
+import com.algorand.algosdk.v2.client.common.QueryData;
+import com.algorand.algosdk.v2.client.common.Response;
+import com.algorand.algosdk.v2.client.model.BlockResponse;
+
+
+/**
+ * /v2/blocks/{round} 
+ */
+public class GetBlock extends Query {
+
+	private Long round;
+
+	/**
+	 * @param round The round from which to fetch block information. 
+	 */
+	public GetBlock(Client client, Long round) {
+		super(client, "get");
+		this.round = round;
+	}
+
+	/**
+	 * Configures whether the response object is JSON or MessagePack encoded. 
+	 */
+	public GetBlock format(String format) {
```

Comment:
This can be an enum:
```
    "format": {
      "enum": [
        "json",
        "msgpack"
      ],
      "type": "string",
      "description": "Configures whether the response object is JSON or MessagePack encoded.",
      "name": "format",
      "in": "query"
    },
```

Reply:
fixed.